node/cmd/guardiand/adminserver.go

```diff
@@ -689,3 +689,40 @@ func (s *nodePrivilegedService) SignExistingVAA(ctx context.Context, req *nodev1
 
 	return &nodev1.SignExistingVAAResponse{Vaa: newVAABytes}, nil
 }
+
+func (s *nodePrivilegedService) DumpRPCs(ctx context.Context, req *nodev1.DumpRPCsRequest) (*nodev1.DumpRPCsResponse, error) {
+	rpcMap := make(map[string]string)
+
+	rpcMap["acalaRPC"] = *acalaRPC
+	rpcMap["algorandIndexerRPC"] = *algorandIndexerRPC
+	rpcMap["algorandAlgodRPC"] = *algorandAlgodRPC
+	rpcMap["aptosRPC"] = *aptosRPC
+	rpcMap["arbitrumRPC"] = *arbitrumRPC
+	rpcMap["auroraRPC"] = *auroraRPC
+	rpcMap["avalancheRPC"] = *avalancheRPC
+	rpcMap["bscRPC"] = *bscRPC
+	rpcMap["celoRPC"] = *celoRPC
+	rpcMap["ethRPC"] = *ethRPC
+	rpcMap["fantomRPC"] = *fantomRPC
+	rpcMap["karuraRPC"] = *karuraRPC
+	rpcMap["klaytnRPC"] = *klaytnRPC
+	rpcMap["moonbeamRPC"] = *moonbeamRPC
+	rpcMap["nearRPC"] = *nearRPC
+	rpcMap["neonRPC"] = *neonRPC
+	rpcMap["oasisRPC"] = *oasisRPC
+	rpcMap["polygonRPC"] = *polygonRPC
+	rpcMap["pythnetRPC"] = *pythnetRPC
```

Comment:
🤔 can handle in a follow up, but isn't there a `pythnetWS` now? wonder if there's a cleaner way to keep these in sync with the flags.